src/App.jsx

```diff
@@ -2,9 +2,9 @@ import React, { useState } from 'react';
 import Title from './Title';
 import styled from 'styled-components';
 import GlobalStyle from './GlobalStyle';
-import { Icon } from './components/Icons';
+import { Icon } from './components/Icon';
 import Input from './components/Input';
-import Button from './components/Button';
+import Button from './components/index';
```

Comment:
Esse import deveria estar com chaves e o index no final não é necessário, dessa forma:
```suggestion
import { Button } from './components';
```